tests/unit_tests/test_engine.py

```
@@ -165,13 +170,13 @@ def test_that_language_switch_has_proper_aria_label_text(test_app):
     )
 
 
-def test_that_page_has_proper_html_lang_attribute(test_app):
+def test_that_page_has_proper_html_lang_attribute(test_app: Engine):
     response = test_app.test_client().get("/")
     soup = BeautifulSoup(response.data, "html.parser")
     assert soup.html and soup.html.get("lang") == "en-GB"
```

Ruff (0.14.10)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)